components/cqrs/object_test.go

```diff
@@ -0,0 +1,24 @@
+package cqrs_test
+
+import (
+	"testing"
+
+	"github.com/ThreeDotsLabs/watermill/components/cqrs"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestObjectName(t *testing.T) {
+	type Object struct{}
+
+	assert.Equal(t, "cqrs_test.Object", cqrs.ObjectName(Object{}))
```

Comment:
How do import aliases affect this? My guess is they should not

Reply:
I'd checked and it doesn't